### ISSUE TITLE
Create Promise<Mocked>

### ISSUE THREAD

**Issue by kellpossible** [NiGhTTraX/strong-mock#321]
How do I create a promise of something that I mocked? I keep getting the following errors when using `Promise.resolve(myMockedObject)`:

```
Error: Didn't expect mock.then([Function anonymous], [Function anonymous]) to be called.
```

**Comment by NiGhTTraX** [comment]
Thanks for raising this, @kellpossible! Fixed in [8.0.1](https://github.com/NiGhTTraX/strong-mock/blob/2319e0783dd647878ed3c32617a5db803b3db454/CHANGELOG.md#801-2023-01-18).

**Comment by kellpossible** [comment]
Thank you!
